settings/base.py

```diff
@@ -139,6 +139,7 @@
         "rest_framework.authentication.SessionAuthentication",
         "oauth2_provider.contrib.rest_framework.OAuth2Authentication",
     ),
+    "DEFAULT_FILTER_BACKENDS": ("django_filters.rest_framework.DjangoFilterBackend",),
```

Comment:
now that we have this we can remove all traces of `filter_backends = (filters.DjangoFilterBackend,)` in the views.